docs/base.md

```diff
@@ -236,10 +308,10 @@ Spawns an Object using a JSON string. Works with [getJSON()](object.md#getjson).
 			* {>>A reference to the object spawned is always passed to callback_function. See the example for how to access it.<<}
 
 ``` Lua
-function onLoad()
+function onObjectDrop(player_color, dropped_object)
```

Comment:
This should be a separate PR.

Also, whilst I understand the motivation for the change, I'd actually rather the example just call the function and not confuse users with potentially unrelated. Newbies can get confused and think the spawning _must_ happen in these callbacks. Best to keep things short and sweet. So if you do submit a PR for this, I reckon just remove the calling context altogether.

Reply:
I'll change this back to onLoad for the purposes of separating the "onLoad is a dumb place to spawn a copy of yourself" issue in to a new PR.